.github/workflows/R-CMD-check.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: macOS-13
     strategy:
       matrix:
-        r-version: ['4.4']
+        r-version: ['4.1']
```

Comment:
Same issue again with workflow getting modified. @daomcgill @RavenMarQ @crepesAlot @beydlern make sure you account for this